src/index.html

```diff
@@ -13,5 +13,93 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <h2>Personal information</h2>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset> <legend>Personal information</legend>
+
+<label>
+  Surname: 
+<input type="text" name="surname" autocomplete="off">  
+</label>
+
+<label>
+  Name:
+<input type="text" name="name" autocomplete="off">  
+</label>
+
+<label>
+  How old are you?
+  <input type="number" name="old" max="100" min="1" value="12">  
+  </label>
+  <label>
+    Full date of birth:
+    <input type="date" name="birth">
+  </label>
+  <label>
+    I accept the term of the agreement
+    <input type="checkbox" name="accept" required>
+  </label>
+</fieldset>
+  
+  
+  <fieldset> <legend>Registration</legend>
+    
+<label>
+  E-mail:
+  <input type="email" name="email" placeholder="email@example.com">
+</label>
+
+<label>
+  Password:
+<input type="password" name="password" minlength="8" maxlength="32" >
+</label>
+  </fieldset>
+  
+  <fieldset> <legend> An intresting facr about you</legend>
+    
+    <label >
+ Do you love cats? 
+ <input type="radio" name="cats"> Yes
+ <input type="radio" name="cats"> No
+    </label>
+    
+    <label>
+What is your favorite color?
+<input type="color" name="color">
+    </label>
+    <label>
+What time do you go to bed?
+<input type="time" name="time">
+    </label>
+    <label>
+What is your favorite brand of cars?
+<select name="car" multiple >
+  <option value="BMW" selected>BMW</option>
+  <option value="Audi">Audi</option>
+  <option value="Lada">Lada</option>
+</select>
+    </label>
+    <label>
+      How do you rate our work?
+      <input type="range">
+          </label>
+  </fieldset>
+  
+  <fieldset> <legend>Additional information</legend>
+    
+    <label >
+      Comments: 
+      <textarea name="feedback" id="" cols="30" rows="10"></textarea>
+    </label>
+    <label>
+      Would you recommend us?
+      <select name="work">
+        <option value="yes" selected>yes</option>
+        <option value="no">no</option>
+      </select>
+    </label>
+  </fieldset>
+  <input type="submit">
+  </form>
   </body>
-</html>
+</html>
```

Comment:
Don't forget to add an empty line at the end
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Not fixed 
![image](https://user-images.githubusercontent.com/96409563/228154205-990f4d00-5010-4791-a8ad-76a004d136da.png)